listenbrainz/webserver/static/js/src/home/Blog.tsx

```diff
@@ -1,23 +1,23 @@
 import React, { useEffect, useState } from "react";
 
-const Blog = () => {
-  const [blogDetails, setBlogDetails] = useState([{}]);
+const Blog = ({ apiUrl }: { apiUrl: string }) => {
+  const [blogDetails, setBlogDetails] = useState<object[]>();
 
   const fetchBlogDetails = async () => {
     try {
-      let response: any = await fetch(
-        `https://public-api.wordpress.com/rest/v1.1/sites/blog.metabrainz.org/posts/`
-      );
-      response = await response.json();
-      const objectArray: any = [];
-      for (let postIndex = 0; postIndex <= 6; postIndex += 1) {
-        const object: any = {};
-        object.id = response.posts[postIndex].ID;
-        object.title = response.posts[postIndex].title;
-        object.link = response.posts[postIndex].URL;
-        objectArray.push(object);
+      let response: any = await fetch(`${apiUrl}/blog-data/`);
```

Comment:
The endpoint is on the website url not the api url so should probably use `${window.location.origin}` instead of api url.

Reply:
fixed